test/check-application

```diff
@@ -132,6 +132,23 @@ class TestApplication(testlib.MachineCase):
         b.click(".modal-dialog div .alert .close")
         b.wait_not_present(".modal-dialog div .alert")
 
+        # delete image busybox that hasn't been used
```

Comment:
You should really consider splitting the testBasic into multiple test functions. testDelete, testAdd etc. It will be easier to maintain! Not a must for this PR, just let's not grow this function more in the future.

Reply:
:+1:

Reply:
Is it iok if I make it in different test files?

Reply:
If you end up creating different file for each feature you 'll end up with dozens of files. I think keeping them in one file under one class but in different function is the way to go for now. But it's not a must for this PR. Feel free to change that in future PRs. :)